src/app/modules/sgid/sgid.service.ts

```diff
@@ -31,31 +31,41 @@ export class SgidService {
     cookieDomain,
     cookieMaxAge,
     cookieMaxAgePreserved,
-    privateKey,
-    publicKey,
+    privateKeyPath,
+    publicKeyPath,
     ...sgidOptions
   }: {
     endpoint: string
     clientId: string
     clientSecret: string
-    privateKey: string
-    publicKey: string
+    privateKeyPath: string
+    publicKeyPath: string
     redirectUri: string
     cookieDomain: string
     cookieMaxAge: number
     cookieMaxAgePreserved: number
   }) {
     this.client = new SgidClient({
       ...sgidOptions,
-      privateKey: fs.readFileSync(privateKey),
+      privateKey: fs.readFileSync(privateKeyPath),
     })
-    this.publicKey = fs.readFileSync(publicKey)
+    this.publicKeyPath = fs.readFileSync(publicKeyPath)
     this.cookieDomain = cookieDomain
     this.cookieMaxAge = cookieMaxAge
     this.cookieMaxAgePreserved = cookieMaxAgePreserved
   }
 
-  createRedirectUrl(state: string): Result<string, SgidCreateRedirectUrlError> {
+  /**
+   * Create a URL to sgID which is used to redirect the user for authentication
+   * @param {String} formId - the form id to redirect to after authentication
```

Comment:
nit: don't need types in JSDoc anymore, the Typescript typings should be the source of truth?